src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1378,6 +1378,11 @@ format layout_optimizer::get_preferred_format(program_node& node) {
             expected = format::get_default_format(input_layout.get_rank());
         }
     } else if (node.is_type<resample>()) {
+        // Resample does not need preferred input format. Propagated format should be selected for both data input and input of fused ops.
+        for (size_t i = 0; i < node.get_dependencies().size(); i++) {
+            node.set_preferred_input_fmt(i, format::any);
+        }
```

Comment:
We aligned planar format is set because of this line. https://github.com/openvinotoolkit/openvino/blob/cad9e8e7d2065697546b5447b27404a2b0f157d4/src/plugins/intel_gpu/src/graph/layout_optimizer.cpp#L1307
Could you check why we should set planar, because we need to set any in here again.